analysis/test_data_utils.py

```diff
@@ -63,6 +48,110 @@ def create_experiment_data():
     ])
 
 
+def test_drop_uniteresting_columns():
+    experiment_df = create_experiment_data()
+    cleaned_df = data_utils.drop_uninteresting_columns(experiment_df)
+
+    assert 'time_started' not in cleaned_df.columns
+
+
+def test_filter_fuzzers():
+    experiment_df = create_experiment_data()
+    fuzzers_to_keep = ['afl']
+    filtered_df = data_utils.filter_fuzzers(experiment_df, fuzzers_to_keep)
+
+    assert filtered_df.fuzzer.unique() == fuzzers_to_keep
+
+
+def test_filter_benchmarks():
+    experiment_df = create_experiment_data()
+    benchmarks_to_keep = ['libpng']
+    filtered_df = data_utils.filter_benchmarks(experiment_df,
+                                               benchmarks_to_keep)
+
+    assert filtered_df.benchmark.unique() == benchmarks_to_keep
+
+
+def test_label_fuzzers_by_experiment():
+    experiment_df = create_experiment_data()
+    labeled_df = data_utils.label_fuzzers_by_experiment(experiment_df)
+
+    expected_fuzzer_names = ['afl-test_experiment', 'libfuzzer-test_experiment']
+    assert labeled_df.fuzzer.unique().tolist() == expected_fuzzer_names
+
+
+def test_benchmark_snapshot():
```

Comment:
Add a comment to this test please. It isn't obvious from the name what is being tested.

Reply:
Done